src/test/fuzz/multiplication_overflow.cpp

```diff
@@ -10,19 +10,33 @@
 #include <string>
 #include <vector>
 
+#if defined(__has_builtin)
```

Comment:
I don't really think we want a reversion here (or in addition), and if we were reverting shouldn't the `__GNUC__` case go away?

In any case, I'd rather we just not define this for MSVC.

Reply:
> I'd rather we just not define this for MSVC.

Thanks! Done.